WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/FeatureConfig/EndToEndIdentityFeatureConfigDecoder.swift

```diff
@@ -30,14 +30,14 @@ struct EndToEndIdentityFeatureConfigDecoder {
 
         return EndToEndIdentityFeatureConfig(
             status: payload.status,
-            acmeDiscoveryURL: payload.config.acmeDiscoveryUrl,
+            acmeDiscoveryURL: payload.config.acmeDiscoveryURL,
             verificationExpiration: payload.config.verificationExpiration
         )
     }
 
     private struct Payload: Decodable {
 
-        let acmeDiscoveryUrl: String?
+        let acmeDiscoveryURL: String?
```

Comment:
Isn't it changing the decoding behaviour? I would assume the json has the key `"acmeDiscoveryUrl"` and might not be decoded correctly anymore?

Reply:
Probably, I hope some tests are failing now.

Reply:
`FeatureConfigEventDecodingTests.testDecodingFeatureConfigUpdateEndToEndIdentityEvent()` is failing 👍 just wanted to raise awareness, maybe not all structs have decoding tests.

Reply:
yup, already fixed, thanks!

Reply:
@markusfassbender nice catch